2017-12-03T21:00:46.131+0900 I CONTROL  [initandlisten] MongoDB starting : pid=9585 port=47019 dbpath=/data/shard1/rs2 64-bit host=kris-ThinkPad-X1-Carbon-5th
2017-12-03T21:00:46.131+0900 I CONTROL  [initandlisten] db version v3.4.10
2017-12-03T21:00:46.131+0900 I CONTROL  [initandlisten] git version: 078f28920cb24de0dd479b5ea6c66c644f6326e9
2017-12-03T21:00:46.131+0900 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten] allocator: tcmalloc
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten] modules: none
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten] build environment:
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten]     distmod: ubuntu1604
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten]     distarch: x86_64
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten]     target_arch: x86_64
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten] options: { net: { port: 47019 }, processManagement: { fork: true }, replication: { replSet: "s1" }, sharding: { clusterRole: "shardsvr" }, storage: { dbPath: "/data/shard1/rs2" }, systemLog: { destination: "file", path: "s1-r2.log" } }
2017-12-03T21:00:46.153+0900 I STORAGE  [initandlisten] 
2017-12-03T21:00:46.153+0900 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2017-12-03T21:00:46.153+0900 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2017-12-03T21:00:46.153+0900 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7269M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),checkpoint=(wait=60,log_size=2GB),statistics_log=(wait=0),
2017-12-03T21:00:46.224+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.224+0900 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2017-12-03T21:00:46.224+0900 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2017-12-03T21:00:46.224+0900 I CONTROL  [initandlisten] ** WARNING: You are running this process as the root user, which is not recommended.
2017-12-03T21:00:46.224+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.225+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.225+0900 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/enabled is 'always'.
2017-12-03T21:00:46.225+0900 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2017-12-03T21:00:46.225+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.225+0900 W SHARDING [initandlisten] Started with --shardsvr, but no shardIdentity document was found on disk in admin.system.version. This most likely means this server has not yet been added to a sharded cluster.
2017-12-03T21:00:46.249+0900 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/shard1/rs2/diagnostic.data'
2017-12-03T21:00:46.270+0900 I REPL     [initandlisten] Did not find local voted for document at startup.
2017-12-03T21:00:46.271+0900 I REPL     [initandlisten] Did not find local replica set configuration document at startup;  NoMatchingDocument: Did not find replica set configuration document in local.system.replset
2017-12-03T21:00:46.271+0900 I NETWORK  [thread1] waiting for connections on port 47019
2017-12-03T21:00:51.354+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37972 #1 (1 connection now open)
2017-12-03T21:00:51.354+0900 I -        [conn1] end connection 127.0.0.1:37972 (1 connection now open)
2017-12-03T21:00:51.354+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37976 #2 (1 connection now open)
2017-12-03T21:00:51.354+0900 I NETWORK  [conn2] received client metadata from 127.0.0.1:37976 conn2: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:00:51.355+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47017
2017-12-03T21:00:51.356+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47017, took 1ms (1 connections now open to localhost:47017)
2017-12-03T21:00:53.357+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37990 #3 (2 connections now open)
2017-12-03T21:00:53.357+0900 I -        [conn3] end connection 127.0.0.1:37990 (2 connections now open)
2017-12-03T21:00:53.397+0900 I REPL     [replExecDBWorker-0] Starting replication storage threads
2017-12-03T21:00:53.471+0900 I REPL     [replication-0] Starting initial sync (attempt 1 of 10)
2017-12-03T21:00:53.471+0900 I REPL     [ReplicationExecutor] New replica set config in use: { _id: "s1", version: 1, protocolVersion: 1, members: [ { _id: 0, host: "localhost:47017", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 1, host: "localhost:47018", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 2, host: "localhost:47019", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 } ], settings: { chainingAllowed: true, heartbeatIntervalMillis: 2000, heartbeatTimeoutSecs: 10, electionTimeoutMillis: 10000, catchUpTimeoutMillis: 60000, getLastErrorModes: {}, getLastErrorDefaults: { w: 1, wtimeout: 0 }, replicaSetId: ObjectId('5a23e7737da32b8ce5ce759f') } }
2017-12-03T21:00:53.471+0900 I REPL     [ReplicationExecutor] This node is localhost:47019 in the config
2017-12-03T21:00:53.471+0900 I REPL     [ReplicationExecutor] transition to STARTUP2
2017-12-03T21:00:53.472+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47018
2017-12-03T21:00:53.472+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state SECONDARY
2017-12-03T21:00:53.473+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47018, took 1ms (1 connections now open to localhost:47018)
2017-12-03T21:00:53.474+0900 I REPL     [ReplicationExecutor] Member localhost:47018 is now in state STARTUP
2017-12-03T21:00:53.475+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37994 #4 (2 connections now open)
2017-12-03T21:00:53.475+0900 I NETWORK  [conn4] received client metadata from 127.0.0.1:37994 conn4: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:00:53.508+0900 I REPL     [replication-0] sync source candidate: localhost:47017
2017-12-03T21:00:53.508+0900 I STORAGE  [replication-0] dropAllDatabasesExceptLocal 1
2017-12-03T21:00:53.508+0900 I REPL     [replication-0] ******
2017-12-03T21:00:53.509+0900 I REPL     [replication-0] creating replication oplog of size: 6116MB...
2017-12-03T21:00:53.529+0900 I STORAGE  [replication-0] Starting WiredTigerRecordStoreThread local.oplog.rs
2017-12-03T21:00:53.529+0900 I STORAGE  [replication-0] The size storer reports that the oplog contains 0 records totaling to 0 bytes
2017-12-03T21:00:53.529+0900 I STORAGE  [replication-0] Scanning the oplog to determine where to place markers for truncation
2017-12-03T21:00:53.629+0900 I REPL     [replication-0] ******
2017-12-03T21:00:53.630+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:00:53.631+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (1 connections now open to localhost:47017)
2017-12-03T21:00:53.633+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:00:53.635+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (2 connections now open to localhost:47017)
2017-12-03T21:00:53.636+0900 I REPL     [replication-1] No need to apply operations. (currently at { : Timestamp 1512302451000|1 })
2017-12-03T21:00:53.637+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending connection to host localhost:47017 due to bad connection status; 1 connections to that host remain open
2017-12-03T21:00:53.637+0900 I REPL     [replication-1] Finished fetching oplog during initial sync: CallbackCanceled: Callback canceled. Last fetched optime and hash: { ts: Timestamp 1512302451000|1, t: -1 }[2865188406503243212]
2017-12-03T21:00:53.637+0900 I REPL     [replication-1] Initial sync attempt finishing up.
2017-12-03T21:00:53.637+0900 I REPL     [replication-1] Initial Sync Attempt Statistics: { failedInitialSyncAttempts: 0, maxFailedInitialSyncAttempts: 10, initialSyncStart: new Date(1512302453471), initialSyncAttempts: [], fetchedMissingDocs: 0, appliedOps: 0, initialSyncOplogStart: Timestamp 1512302451000|1, initialSyncOplogEnd: Timestamp 1512302451000|1, databases: { databasesCloned: 0 } }
2017-12-03T21:00:53.645+0900 I REPL     [replication-1] initial sync done; took 0s.
2017-12-03T21:00:53.645+0900 I REPL     [replication-1] Starting replication fetcher thread
2017-12-03T21:00:53.645+0900 I REPL     [replication-1] Starting replication applier thread
2017-12-03T21:00:53.645+0900 I REPL     [replication-1] Starting replication reporter thread
2017-12-03T21:00:53.646+0900 I REPL     [rsSync] transition to RECOVERING
2017-12-03T21:00:53.646+0900 I REPL     [rsBackgroundSync] could not find member to sync from
2017-12-03T21:00:53.649+0900 I REPL     [ReplicationExecutor] Member localhost:47018 is now in state STARTUP2
2017-12-03T21:00:53.649+0900 I REPL     [rsSync] transition to SECONDARY
2017-12-03T21:00:58.650+0900 I REPL     [ReplicationExecutor] Member localhost:47018 is now in state SECONDARY
2017-12-03T21:01:03.651+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state PRIMARY
2017-12-03T21:01:08.653+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:47017
2017-12-03T21:01:08.654+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:01:08.655+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (2 connections now open to localhost:47017)
2017-12-03T21:01:08.661+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38094 #5 (3 connections now open)
2017-12-03T21:01:08.661+0900 I NETWORK  [conn5] received client metadata from 127.0.0.1:38094 conn5: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:01:08.662+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38096 #6 (4 connections now open)
2017-12-03T21:01:08.663+0900 I NETWORK  [conn6] received client metadata from 127.0.0.1:38096 conn6: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.738+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38170 #7 (5 connections now open)
2017-12-03T21:02:10.738+0900 I NETWORK  [conn7] received client metadata from 127.0.0.1:38170 conn7: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.799+0900 I INDEX    [repl writer worker 1] build index on: admin.system.version properties: { v: 2, key: { version: 1 }, name: "incompatible_with_version_32", ns: "admin.system.version" }
2017-12-03T21:02:10.799+0900 I INDEX    [repl writer worker 1] 	 building index using bulk method; build may temporarily use up to 500 megabytes of RAM
2017-12-03T21:02:10.801+0900 I INDEX    [repl writer worker 1] build index done.  scanned 0 total records. 0 secs
2017-12-03T21:02:10.803+0900 I COMMAND  [repl writer worker 2] setting featureCompatibilityVersion to 3.4
2017-12-03T21:02:10.809+0900 I SHARDING [repl writer worker 13] initializing sharding state with: { _id: "shardIdentity", configsvrConnectionString: "csReplSet/localhost:57040,localhost:57041,localhost:57042", shardName: "s1", clusterId: ObjectId('5a23e786bf51d32cbcc946a4') }
2017-12-03T21:02:10.809+0900 I SHARDING [repl writer worker 13] first cluster operation detected, adding sharding hook to enable versioning and authentication to remote servers
2017-12-03T21:02:10.809+0900 I NETWORK  [repl writer worker 13] Starting new replica set monitor for csReplSet/localhost:57040,localhost:57041,localhost:57042
2017-12-03T21:02:10.810+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57040 (1 connections now open to localhost:57040 with a 5 second timeout)
2017-12-03T21:02:10.812+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57042 (1 connections now open to localhost:57042 with a 5 second timeout)
2017-12-03T21:02:10.813+0900 I SHARDING [thread2] creating distributed lock ping thread for process kris-ThinkPad-X1-Carbon-5th:47019:1512302530:-41758715004048657 (sleeping for 30000ms)
2017-12-03T21:02:10.813+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-03T21:02:10.813+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-03T21:02:10.814+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-03T21:02:10.814+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (2 connections now open to localhost:57040)
2017-12-03T21:02:10.816+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 3ms (2 connections now open to localhost:57040)
2017-12-03T21:02:10.819+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s0/localhost:37017,localhost:37018,localhost:37019
2017-12-03T21:02:10.819+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s1/localhost:47017,localhost:47018,localhost:47019
2017-12-03T21:02:10.820+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-03T21:02:10.821+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37019 (1 connections now open to localhost:37019 with a 5 second timeout)
2017-12-03T21:02:10.822+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37018 (1 connections now open to localhost:37018 with a 5 second timeout)
2017-12-03T21:02:10.823+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47017 (1 connections now open to localhost:47017 with a 5 second timeout)
2017-12-03T21:02:10.827+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: LockStateChangeFailed: findAndModify query predicate didn't match any lock document
2017-12-03T21:02:10.847+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38208 #8 (6 connections now open)
2017-12-03T21:02:10.847+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38236 #9 (7 connections now open)
2017-12-03T21:02:10.849+0900 I NETWORK  [conn8] received client metadata from 127.0.0.1:38208 conn8: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.851+0900 I NETWORK  [conn9] received client metadata from 127.0.0.1:38236 conn9: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.853+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47019 (1 connections now open to localhost:47019 with a 5 second timeout)
2017-12-03T21:02:10.857+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47018 (1 connections now open to localhost:47018 with a 5 second timeout)
2017-12-03T21:02:10.867+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38246 #10 (8 connections now open)
2017-12-03T21:02:10.868+0900 I NETWORK  [conn10] received client metadata from 127.0.0.1:38246 conn10: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.880+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38252 #11 (9 connections now open)
2017-12-03T21:02:10.881+0900 I NETWORK  [conn11] received client metadata from 127.0.0.1:38252 conn11: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.930+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38304 #12 (10 connections now open)
2017-12-03T21:02:10.930+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38306 #13 (11 connections now open)
2017-12-03T21:02:10.930+0900 I NETWORK  [conn13] received client metadata from 127.0.0.1:38306 conn13: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.930+0900 I NETWORK  [conn12] received client metadata from 127.0.0.1:38304 conn12: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.992+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38330 #14 (12 connections now open)
2017-12-03T21:02:10.992+0900 I NETWORK  [conn14] received client metadata from 127.0.0.1:38330 conn14: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:11.204+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38352 #15 (13 connections now open)
2017-12-03T21:02:11.204+0900 I NETWORK  [conn15] received client metadata from 127.0.0.1:38352 conn15: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:27.451+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38376 #16 (14 connections now open)
2017-12-03T21:02:27.451+0900 I NETWORK  [conn16] received client metadata from 127.0.0.1:38376 conn16: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:27.607+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38394 #17 (15 connections now open)
2017-12-03T21:02:27.607+0900 I NETWORK  [conn17] received client metadata from 127.0.0.1:38394 conn17: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.678+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38428 #18 (16 connections now open)
2017-12-03T21:02:40.679+0900 I NETWORK  [conn18] received client metadata from 127.0.0.1:38428 conn18: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.682+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38434 #19 (17 connections now open)
2017-12-03T21:02:40.682+0900 I NETWORK  [conn19] received client metadata from 127.0.0.1:38434 conn19: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.820+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T21:02:40.821+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-03T21:02:40.822+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s2/localhost:57017,localhost:57018,localhost:57019
2017-12-03T21:02:40.825+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-03T21:02:40.828+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57018 (1 connections now open to localhost:57018 with a 5 second timeout)
2017-12-03T21:02:40.830+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57019 (1 connections now open to localhost:57019 with a 5 second timeout)
2017-12-03T21:03:10.817+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Ending idle connection to host localhost:57040 because the pool meets constraints; 1 connections to that host remain open
2017-12-03T21:03:10.823+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T21:03:10.824+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-03T21:12:09.497+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:12:09.498+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-03T21:13:09.497+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T21:23:09.513+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:23:09.515+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-03T21:24:09.514+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T21:32:29.525+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:32:29.527+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-03T21:33:29.526+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T21:34:09.531+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:34:09.533+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-03T21:35:09.532+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T21:44:24.546+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:44:24.549+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 3ms (3 connections now open to localhost:47017)
2017-12-03T21:45:24.547+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T21:58:10.941+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T21:58:10.943+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-03T22:20:40.996+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T22:20:40.998+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-03T22:27:11.036+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T22:27:11.037+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-03T22:33:11.048+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T22:33:11.049+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-03T22:43:29.644+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T22:43:29.645+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-03T22:44:29.645+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T22:58:29.674+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T22:58:29.675+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-03T22:59:29.675+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T23:09:11.125+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T23:09:11.126+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-03T23:25:19.728+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T23:25:19.729+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-03T23:26:19.729+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T23:27:41.168+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T23:27:41.169+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-03T23:32:41.181+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T23:32:41.182+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-03T23:44:41.211+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T23:44:41.212+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-04T00:13:29.813+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T00:13:29.814+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T00:14:29.813+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T00:19:19.823+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T00:19:19.824+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-04T00:20:19.824+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T00:23:49.836+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T00:23:49.840+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 5ms (3 connections now open to localhost:47017)
2017-12-04T00:24:49.836+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T00:27:41.317+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T00:27:41.318+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T00:50:41.375+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T00:50:41.378+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 3ms (1 connections now open to localhost:57041)
2017-12-04T01:03:09.905+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:40778 #20 (18 connections now open)
2017-12-04T01:03:09.905+0900 I NETWORK  [conn20] received client metadata from 127.0.0.1:40778 conn20: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T01:04:09.905+0900 I -        [conn6] end connection 127.0.0.1:38096 (18 connections now open)
2017-12-04T01:09:19.920+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T01:09:19.921+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-04T01:10:19.921+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T01:11:41.429+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T01:11:41.430+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-04T01:21:14.940+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T01:21:14.942+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T01:22:14.941+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T01:24:41.464+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T01:24:41.466+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T01:24:49.946+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T01:24:49.947+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T01:25:49.946+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T01:28:59.955+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T01:28:59.956+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-04T01:29:59.956+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T01:39:41.504+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T01:39:41.506+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T01:43:49.992+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T01:43:49.994+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T01:44:49.994+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T01:47:29.990+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T01:47:29.992+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T01:48:29.991+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T01:52:50.015+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T01:52:50.018+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 3ms (3 connections now open to localhost:47017)
2017-12-04T01:53:50.019+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T02:14:20.055+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T02:14:20.059+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 4ms (3 connections now open to localhost:47017)
2017-12-04T02:15:20.058+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T02:19:11.603+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T02:19:11.605+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T02:35:20.095+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T02:35:20.104+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 9ms (3 connections now open to localhost:47017)
2017-12-04T02:36:20.097+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T02:39:10.104+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T02:39:10.106+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T02:40:40.105+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T03:06:25.159+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T03:06:25.161+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T03:07:25.160+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T03:07:50.163+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T03:07:50.165+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T03:08:50.164+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T03:13:00.171+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T03:13:00.172+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-04T03:14:00.171+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T03:22:11.763+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T03:22:11.763+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-04T03:22:50.193+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T03:22:50.197+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 4ms (3 connections now open to localhost:47017)
2017-12-04T03:23:50.194+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T03:30:40.209+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T03:30:40.211+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T03:31:40.210+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T03:54:41.845+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T03:54:41.849+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 4ms (1 connections now open to localhost:57041)
2017-12-04T03:56:25.256+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T03:56:25.257+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-04T03:57:25.256+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T04:08:11.880+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T04:08:11.882+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T04:08:41.882+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T04:08:41.883+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-04T04:23:50.318+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T04:23:50.319+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-04T04:25:10.311+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T04:45:50.355+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T04:45:50.356+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-04T04:46:50.356+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T04:49:11.984+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T04:49:11.986+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 3ms (1 connections now open to localhost:57041)
2017-12-04T04:57:30.382+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T04:57:30.384+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T04:58:30.383+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T05:02:42.032+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T05:02:42.033+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T05:20:42.077+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T05:20:42.078+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T05:33:12.108+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T05:33:12.110+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T05:45:40.479+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T05:45:40.481+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T05:46:40.480+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T05:46:45.474+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T05:46:45.476+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T05:47:45.475+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T05:47:50.479+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T05:47:50.480+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-04T05:48:50.480+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T05:59:42.177+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T05:59:42.179+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 3ms (1 connections now open to localhost:57042)
2017-12-04T06:19:05.542+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T06:19:05.544+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T06:20:05.543+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T06:35:12.271+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T06:35:12.273+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T06:42:40.590+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T06:42:40.595+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 4ms (3 connections now open to localhost:47017)
2017-12-04T06:43:50.590+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T06:48:12.306+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T06:48:12.307+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-04T07:07:10.634+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T07:07:10.635+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-04T07:08:10.635+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T07:29:12.414+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T07:29:12.417+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 3ms (1 connections now open to localhost:57041)
2017-12-04T07:56:50.728+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T07:56:50.733+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 5ms (3 connections now open to localhost:47017)
2017-12-04T07:57:50.733+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T08:10:12.518+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T08:10:12.520+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T08:13:25.761+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T08:13:25.763+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-04T15:41:54.180+0900 I -        [conn5] end connection 127.0.0.1:38094 (17 connections now open)
2017-12-04T15:41:54.180+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T15:41:54.196+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T15:41:54.196+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending connection to host localhost:47017 due to bad connection status; 0 connections to that host remain open
2017-12-04T15:41:54.196+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T15:41:54.180+0900 I REPL     [ReplicationExecutor] Starting an election, since we've seen no PRIMARY in the past 10000ms
2017-12-04T15:41:54.196+0900 I REPL     [ReplicationExecutor] conducting a dry run election to see if we could be elected
2017-12-04T15:41:54.196+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47017
2017-12-04T15:41:54.183+0900 I COMMAND  [conn20] command local.oplog.rs command: getMore { getMore: 34604852486, collection: "oplog.rs", maxTimeMS: 5000, term: 1, lastKnownCommittedOpTime: { ts: Timestamp 1512342825000|1, t: 1 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512302451000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 1 } planSummary: COLLSCAN cursorid:34604852486 keysExamined:0 docsExamined:0 numYields:0 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 4 } }, Database: { acquireCount: { r: 2 } }, oplog: { acquireCount: { r: 2 } } } protocol:op_command 26883418ms
2017-12-04T15:41:54.196+0900 I -        [conn20] end connection 127.0.0.1:40778 (16 connections now open)
2017-12-04T15:41:54.197+0900 I REPL     [replication-8] Restarting oplog query due to error: ExceededTimeLimit: Operation timed out, request was RemoteCommand 154244 -- target:localhost:47017 db:local expDate:2017-12-04T08:14:55.766+0900 cmd:{ getMore: 18148934450, collection: "oplog.rs", maxTimeMS: 5000, term: 1, lastKnownCommittedOpTime: { ts: Timestamp 1512342825000|1, t: 1 } }. Last fetched optime (with hash): { ts: Timestamp 1512342825000|1, t: 1 }[5334871594228210341]. Restarts remaining: 3
2017-12-04T15:41:54.197+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-04T15:41:54.197+0900 I REPL     [replication-8] Scheduled new oplog query Fetcher source: localhost:47017 database: local query: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512342825000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 1, readConcern: { afterOpTime: { ts: Timestamp 1512342825000|1, t: 1 } } } query metadata: { $replData: 1, $oplogQueryData: 1, $ssm: { $secondaryOk: true } } active: 1 timeout: 65000ms shutting down?: 0 first: 1 firstCommandScheduler: RemoteCommandRetryScheduler request: RemoteCommand 154264 -- target:localhost:47017 db:local cmd:{ find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512342825000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 1, readConcern: { afterOpTime: { ts: Timestamp 1512342825000|1, t: 1 } } } active: 1 callbackHandle.valid: 1 callbackHandle.cancelled: 0 attempt: 1 retryPolicy: RetryPolicyImpl maxAttempts: 1 maxTimeMillis: -1ms
2017-12-04T15:41:54.197+0900 I REPL     [ReplicationExecutor] VoteRequester(term 1 dry run) received a yes vote from localhost:47018; response message: { term: 1, voteGranted: true, reason: "", ok: 1.0 }
2017-12-04T15:41:54.197+0900 I REPL     [ReplicationExecutor] dry election run succeeded, running for election
2017-12-04T15:41:54.202+0900 I REPL     [ReplicationExecutor] VoteRequester(term 2) received a no vote from localhost:47018 with reason "already voted for another candidate this term"; response message: { term: 2, voteGranted: false, reason: "already voted for another candidate this term", ok: 1.0 }
2017-12-04T15:41:54.205+0900 I REPL     [ReplicationExecutor] VoteRequester(term 2) received a no vote from localhost:47017 with reason "candidate's data is staler than mine"; response message: { term: 2, voteGranted: false, reason: "candidate's data is staler than mine", ok: 1.0 }
2017-12-04T15:41:54.206+0900 I REPL     [ReplicationExecutor] not becoming primary, we received insufficient votes
2017-12-04T15:41:54.209+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42278 #21 (16 connections now open)
2017-12-04T15:41:54.209+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42280 #22 (17 connections now open)
2017-12-04T15:41:54.209+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42286 #23 (18 connections now open)
2017-12-04T15:41:54.210+0900 I NETWORK  [conn23] received client metadata from 127.0.0.1:42286 conn23: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:41:54.210+0900 I NETWORK  [conn21] received client metadata from 127.0.0.1:42278 conn21: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:41:54.210+0900 I NETWORK  [conn22] received client metadata from 127.0.0.1:42280 conn22: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:41:54.211+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57040 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T15:41:54.211+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T15:41:54.211+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47017, took 15ms (2 connections now open to localhost:47017)
2017-12-04T15:41:54.211+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 14ms (2 connections now open to localhost:47017)
2017-12-04T15:41:54.212+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 16ms (2 connections now open to localhost:47017)
2017-12-04T15:41:54.223+0900 I REPL     [replication-7] Choosing new sync source because our current sync source, localhost:47017, has an OpTime ({ ts: Timestamp 1512369714000|1, t: 1 }) which is not ahead of ours ({ ts: Timestamp 1512369714000|1, t: 1 }), it does not have a sync source, and it's not the primary (sync source does not know the primary)
2017-12-04T15:41:54.223+0900 I REPL     [replication-7] Canceling oplog query due to OplogQueryMetadata. We have to choose a new sync source. Current source: localhost:47017, OpTime { ts: Timestamp 1512369714000|1, t: 1 }, its sync source index:-1
2017-12-04T15:41:54.223+0900 W REPL     [rsBackgroundSync] Fetcher stopped querying remote oplog with error: InvalidSyncSource: sync source localhost:47017 (config version: 1; last applied optime: { ts: Timestamp 1512369714000|1, t: 1 }; sync source index: -1; primary index: -1) is no longer valid
2017-12-04T15:41:54.223+0900 I REPL     [rsBackgroundSync] could not find member to sync from
2017-12-04T15:41:54.224+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state SECONDARY
2017-12-04T15:41:54.226+0900 I REPL     [SyncSourceFeedback] SyncSourceFeedback error sending update to localhost:47017: InvalidSyncSource: Sync source was cleared. Was localhost:47017
2017-12-04T15:42:04.617+0900 I REPL     [ReplicationExecutor] Starting an election, since we've seen no PRIMARY in the past 10000ms
2017-12-04T15:42:04.617+0900 I REPL     [ReplicationExecutor] conducting a dry run election to see if we could be elected
2017-12-04T15:42:04.620+0900 I REPL     [ReplicationExecutor] VoteRequester(term 2 dry run) received a yes vote from localhost:47017; response message: { term: 2, voteGranted: true, reason: "", ok: 1.0 }
2017-12-04T15:42:04.620+0900 I REPL     [ReplicationExecutor] dry election run succeeded, running for election
2017-12-04T15:42:04.636+0900 I REPL     [ReplicationExecutor] VoteRequester(term 3) received a yes vote from localhost:47017; response message: { term: 3, voteGranted: true, reason: "", ok: 1.0 }
2017-12-04T15:42:04.636+0900 I REPL     [ReplicationExecutor] election succeeded, assuming primary role in term 3
2017-12-04T15:42:04.637+0900 I REPL     [ReplicationExecutor] transition to PRIMARY
2017-12-04T15:42:04.637+0900 I REPL     [ReplicationExecutor] Entering primary catch-up mode.
2017-12-04T15:42:04.637+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47018
2017-12-04T15:42:04.638+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending connection to host localhost:47018 due to bad connection status; 1 connections to that host remain open
2017-12-04T15:42:04.641+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47018, took 4ms (1 connections now open to localhost:47018)
2017-12-04T15:42:04.642+0900 I REPL     [ReplicationExecutor] Caught up to the latest optime known via heartbeats after becoming primary.
2017-12-04T15:42:04.642+0900 I REPL     [ReplicationExecutor] Exited primary catch-up mode.
2017-12-04T15:42:06.224+0900 I REPL     [rsSync] transition to primary complete; database writes are now permitted
2017-12-04T15:42:07.242+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42360 #24 (19 connections now open)
2017-12-04T15:42:07.242+0900 I NETWORK  [conn24] received client metadata from 127.0.0.1:42360 conn24: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:42:10.464+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T15:42:10.466+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T15:42:10.837+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:57040)
2017-12-04T15:42:10.839+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57040 (1 connections now open to localhost:57040 with a 5 second timeout)
2017-12-04T15:42:10.841+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:57017)
2017-12-04T15:42:10.842+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-04T15:42:10.844+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:37017)
2017-12-04T15:42:10.845+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-04T15:42:10.847+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:47017)
2017-12-04T15:42:10.849+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47017 (1 connections now open to localhost:47017 with a 5 second timeout)
2017-12-04T15:42:12.182+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42462 #25 (20 connections now open)
2017-12-04T15:42:12.182+0900 I NETWORK  [conn25] received client metadata from 127.0.0.1:42462 conn25: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:42:12.185+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42464 #26 (21 connections now open)
2017-12-04T15:42:12.185+0900 I NETWORK  [conn26] received client metadata from 127.0.0.1:42464 conn26: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:42:54.203+0900 I -        [conn4] end connection 127.0.0.1:37994 (21 connections now open)
2017-12-04T15:42:54.206+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T15:42:54.222+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T15:44:10.470+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T15:44:10.471+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T15:45:05.734+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42506 #27 (21 connections now open)
2017-12-04T15:45:05.735+0900 I NETWORK  [conn27] received client metadata from 127.0.0.1:42506 conn27: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:45:05.746+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42508 #28 (22 connections now open)
2017-12-04T15:45:05.746+0900 I NETWORK  [conn28] received client metadata from 127.0.0.1:42508 conn28: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-1", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:45:51.173+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:42816 #29 (23 connections now open)
2017-12-04T15:45:51.173+0900 I NETWORK  [conn29] received client metadata from 127.0.0.1:42816 conn29: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-3", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:48:00.200+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43870 #30 (24 connections now open)
2017-12-04T15:48:00.203+0900 I NETWORK  [conn30] received client metadata from 127.0.0.1:43870 conn30: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-2", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:48:07.358+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:43934 #31 (25 connections now open)
2017-12-04T15:48:07.358+0900 I NETWORK  [conn31] received client metadata from 127.0.0.1:43934 conn31: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-0", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:50:05.747+0900 I -        [conn28] end connection 127.0.0.1:42508 (25 connections now open)
2017-12-04T15:50:51.176+0900 I -        [conn29] end connection 127.0.0.1:42816 (24 connections now open)
2017-12-04T15:52:06.688+0900 I COMMAND  [conn21] command local.oplog.rs command: getMore { getMore: 35554011462, collection: "oplog.rs", maxTimeMS: 5000, term: 3, lastKnownCommittedOpTime: { ts: Timestamp 1512370316000|1, t: 3 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512369714000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 3, readConcern: { afterOpTime: { ts: Timestamp 1512369714000|1, t: 1 } } } planSummary: COLLSCAN cursorid:35554011462 keysExamined:0 docsExamined:0 numYields:1 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 6 } }, Database: { acquireCount: { r: 3 } }, oplog: { acquireCount: { r: 3 } } } protocol:op_command 5438ms
2017-12-04T15:52:06.700+0900 I COMMAND  [conn25] command local.oplog.rs command: getMore { getMore: 34539207195, collection: "oplog.rs", maxTimeMS: 5000, term: 3, lastKnownCommittedOpTime: { ts: Timestamp 1512370316000|1, t: 3 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512369714000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 3, readConcern: { afterOpTime: { ts: Timestamp 1512369714000|1, t: 1 } } } planSummary: COLLSCAN cursorid:34539207195 keysExamined:0 docsExamined:0 numYields:0 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 4 } }, Database: { acquireCount: { r: 2 } }, oplog: { acquireCount: { r: 2 } } } protocol:op_command 5450ms
2017-12-04T15:56:22.459+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:44438 #32 (24 connections now open)
2017-12-04T15:56:22.459+0900 I NETWORK  [conn32] received client metadata from 127.0.0.1:44438 conn32: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-1", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:56:46.913+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:44648 #33 (25 connections now open)
2017-12-04T15:56:46.914+0900 I NETWORK  [conn33] received client metadata from 127.0.0.1:44648 conn33: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-3", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:59:53.353+0900 I -        [conn31] end connection 127.0.0.1:43934 (25 connections now open)
2017-12-04T15:59:59.603+0900 I -        [conn30] end connection 127.0.0.1:43870 (24 connections now open)
2017-12-04T16:00:38.099+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:46572 #34 (24 connections now open)
2017-12-04T16:00:38.100+0900 I NETWORK  [conn34] received client metadata from 127.0.0.1:46572 conn34: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-0", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:01:22.460+0900 I -        [conn32] end connection 127.0.0.1:44438 (24 connections now open)
2017-12-04T16:01:46.916+0900 I -        [conn33] end connection 127.0.0.1:44648 (23 connections now open)
2017-12-04T16:04:40.552+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:48544 #35 (23 connections now open)
2017-12-04T16:04:40.552+0900 I NETWORK  [conn35] received client metadata from 127.0.0.1:48544 conn35: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-2", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:06:41.256+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T16:06:41.257+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-04T16:13:30.549+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:52946 #36 (24 connections now open)
2017-12-04T16:13:30.549+0900 I NETWORK  [conn36] received client metadata from 127.0.0.1:52946 conn36: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-3", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:13:57.292+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:53170 #37 (25 connections now open)
2017-12-04T16:13:57.292+0900 I NETWORK  [conn37] received client metadata from 127.0.0.1:53170 conn37: { driver: { name: "NetworkInterfaceASIO-TaskExecutorPool-1", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:17:32.410+0900 I -        [conn34] end connection 127.0.0.1:46572 (25 connections now open)
2017-12-04T16:17:38.934+0900 I -        [conn35] end connection 127.0.0.1:48544 (24 connections now open)
2017-12-04T16:19:41.282+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T16:19:41.283+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T16:21:34.208+0900 I -        [conn36] end connection 127.0.0.1:52946 (23 connections now open)
2017-12-04T16:21:36.102+0900 I -        [conn37] end connection 127.0.0.1:53170 (22 connections now open)
2017-12-04T16:45:25.183+0900 I COMMAND  [conn26] command local.oplog.rs command: getMore { getMore: 34539207195, collection: "oplog.rs", maxTimeMS: 5000, term: 3, lastKnownCommittedOpTime: { ts: Timestamp 1512372446000|1, t: 3 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512369714000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 3, readConcern: { afterOpTime: { ts: Timestamp 1512369714000|1, t: 1 } } } planSummary: COLLSCAN cursorid:34539207195 keysExamined:0 docsExamined:0 numYields:1 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 6 } }, Database: { acquireCount: { r: 3 } }, oplog: { acquireCount: { r: 3 } } } protocol:op_command 1073426ms
2017-12-04T16:45:25.183+0900 I COMMAND  [conn21] command local.oplog.rs command: getMore { getMore: 35554011462, collection: "oplog.rs", maxTimeMS: 5000, term: 3, lastKnownCommittedOpTime: { ts: Timestamp 1512372446000|1, t: 3 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512369714000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 3, readConcern: { afterOpTime: { ts: Timestamp 1512369714000|1, t: 1 } } } planSummary: COLLSCAN cursorid:35554011462 keysExamined:0 docsExamined:1 numYields:1 nreturned:1 reslen:662 locks:{ Global: { acquireCount: { r: 6 } }, Database: { acquireCount: { r: 3 } }, oplog: { acquireCount: { r: 3 } } } protocol:op_command 1073427ms
2017-12-04T16:45:25.185+0900 I REPL     [ReplicationExecutor] can't see a majority of the set, relinquishing primary
2017-12-04T16:45:25.185+0900 I REPL     [ReplicationExecutor] Stepping down from primary in response to heartbeat
2017-12-04T16:45:25.185+0900 I REPL     [replExecDBWorker-0] transition to SECONDARY
2017-12-04T16:45:25.186+0900 I NETWORK  [replExecDBWorker-0] legacy transport layer closing all connections
2017-12-04T16:45:25.186+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 26
2017-12-04T16:45:25.186+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 25
2017-12-04T16:45:25.186+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 24
2017-12-04T16:45:25.186+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 23
2017-12-04T16:45:25.186+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 22
2017-12-04T16:45:25.186+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 21
2017-12-04T16:45:25.186+0900 I NETWORK  [replExecDBWorker-0] Skip closing connection for connection # 2
2017-12-04T16:45:25.186+0900 I -        [conn11] end connection 127.0.0.1:38252 (21 connections now open)
2017-12-04T16:45:25.186+0900 I -        [conn7] end connection 127.0.0.1:38170 (20 connections now open)
2017-12-04T16:45:25.188+0900 I -        [conn10] end connection 127.0.0.1:38246 (19 connections now open)
2017-12-04T16:45:25.188+0900 I -        [conn16] end connection 127.0.0.1:38376 (18 connections now open)
2017-12-04T16:45:25.188+0900 I -        [conn19] end connection 127.0.0.1:38434 (17 connections now open)
2017-12-04T16:45:25.188+0900 I -        [conn18] end connection 127.0.0.1:38428 (16 connections now open)
2017-12-04T16:45:25.188+0900 I -        [conn8] end connection 127.0.0.1:38208 (15 connections now open)
2017-12-04T16:45:25.188+0900 I -        [conn14] end connection 127.0.0.1:38330 (14 connections now open)
2017-12-04T16:45:25.188+0900 I -        [conn12] end connection 127.0.0.1:38304 (13 connections now open)
2017-12-04T16:45:25.188+0900 I -        [conn15] end connection 127.0.0.1:38352 (12 connections now open)
2017-12-04T16:45:25.189+0900 I REPL     [ReplicationExecutor] Member localhost:47018 is now in state SECONDARY
2017-12-04T16:45:25.190+0900 I -        [conn9] end connection 127.0.0.1:38236 (11 connections now open)
2017-12-04T16:45:25.190+0900 I -        [conn13] end connection 127.0.0.1:38306 (10 connections now open)
2017-12-04T16:45:25.190+0900 I -        [conn17] end connection 127.0.0.1:38394 (9 connections now open)
2017-12-04T16:45:25.190+0900 I -        [conn27] end connection 127.0.0.1:42506 (8 connections now open)
2017-12-04T16:45:25.194+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state SECONDARY
2017-12-04T16:45:25.219+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1094 secs, remote host 127.0.0.1:37018)
2017-12-04T16:45:25.219+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1094 secs, remote host 127.0.0.1:47019)
2017-12-04T16:45:25.219+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1094 secs, remote host 127.0.0.1:57019)
2017-12-04T16:45:25.256+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57041 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T16:45:25.256+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T16:45:27.350+0900 I -        [conn25] end connection 127.0.0.1:42462 (7 connections now open)
2017-12-04T16:45:31.203+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60224 #38 (7 connections now open)
2017-12-04T16:45:31.204+0900 I NETWORK  [conn38] received client metadata from 127.0.0.1:60224 conn38: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:31.892+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T16:45:31.894+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-04T16:45:32.014+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60262 #39 (8 connections now open)
2017-12-04T16:45:32.014+0900 I NETWORK  [conn39] received client metadata from 127.0.0.1:60262 conn39: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.029+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60280 #40 (9 connections now open)
2017-12-04T16:45:32.029+0900 I NETWORK  [conn40] received client metadata from 127.0.0.1:60280 conn40: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.030+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60282 #41 (10 connections now open)
2017-12-04T16:45:32.030+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60284 #42 (11 connections now open)
2017-12-04T16:45:32.030+0900 I NETWORK  [conn41] received client metadata from 127.0.0.1:60282 conn41: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.030+0900 I NETWORK  [conn42] received client metadata from 127.0.0.1:60284 conn42: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.303+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 7 secs, remote host 127.0.0.1:57041)
2017-12-04T16:45:32.305+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-04T16:45:32.307+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57019 (1 connections now open to localhost:57019 with a 5 second timeout)
2017-12-04T16:45:32.309+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60312 #43 (12 connections now open)
2017-12-04T16:45:32.310+0900 I NETWORK  [conn43] received client metadata from 127.0.0.1:60312 conn43: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.310+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37018 (1 connections now open to localhost:37018 with a 5 second timeout)
2017-12-04T16:45:32.312+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60318 #44 (13 connections now open)
2017-12-04T16:45:32.312+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60320 #45 (14 connections now open)
2017-12-04T16:45:32.312+0900 I NETWORK  [conn44] received client metadata from 127.0.0.1:60318 conn44: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.313+0900 I NETWORK  [conn45] received client metadata from 127.0.0.1:60320 conn45: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.313+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47019 (1 connections now open to localhost:47019 with a 5 second timeout)
2017-12-04T16:45:32.314+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s1
2017-12-04T16:45:32.333+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60328 #46 (15 connections now open)
2017-12-04T16:45:32.333+0900 I NETWORK  [conn46] received client metadata from 127.0.0.1:60328 conn46: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.333+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60332 #47 (16 connections now open)
2017-12-04T16:45:32.334+0900 I NETWORK  [conn47] received client metadata from 127.0.0.1:60332 conn47: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.341+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60342 #48 (17 connections now open)
2017-12-04T16:45:32.341+0900 I NETWORK  [conn48] received client metadata from 127.0.0.1:60342 conn48: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:40.192+0900 I REPL     [ReplicationExecutor] Member localhost:47018 is now in state PRIMARY
2017-12-04T16:45:40.194+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:47018
2017-12-04T16:45:40.194+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47018
2017-12-04T16:45:40.195+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47018, took 1ms (1 connections now open to localhost:47018)
2017-12-04T16:45:40.196+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47018
2017-12-04T16:45:40.197+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47018, took 1ms (2 connections now open to localhost:47018)
2017-12-04T16:45:48.931+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60490 #49 (18 connections now open)
2017-12-04T16:45:48.931+0900 I NETWORK  [conn49] received client metadata from 127.0.0.1:60490 conn49: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:49.060+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60498 #50 (19 connections now open)
2017-12-04T16:45:49.060+0900 I NETWORK  [conn50] received client metadata from 127.0.0.1:60498 conn50: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:49.452+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:60510 #51 (20 connections now open)
2017-12-04T16:45:49.453+0900 I NETWORK  [conn51] received client metadata from 127.0.0.1:60510 conn51: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:55.257+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T16:45:55.259+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T16:46:25.219+0900 I -        [conn22] end connection 127.0.0.1:42280 (20 connections now open)
2017-12-04T16:50:25.216+0900 I -        [conn24] end connection 127.0.0.1:42360 (19 connections now open)
2017-12-04T16:50:25.252+0900 I -        [conn26] end connection 127.0.0.1:42464 (18 connections now open)
2017-12-04T16:50:30.219+0900 I -        [conn21] end connection 127.0.0.1:42278 (17 connections now open)
2017-12-04T16:50:41.225+0900 I -        [conn38] end connection 127.0.0.1:60224 (16 connections now open)
2017-12-04T17:42:33.033+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T17:42:33.036+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 3ms (1 connections now open to localhost:57041)
2017-12-04T17:49:33.050+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T17:49:33.052+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-04T17:50:04.189+0900 I INDEX    [repl writer worker 1] build index on: mongomart.item properties: { v: 2, key: { _fts: "text", _ftsx: 1 }, name: "title_text_slogan_text_description_text", ns: "mongomart.item", weights: { description: 1, slogan: 1, title: 1 }, default_language: "english", language_override: "language", textIndexVersion: 3 }
2017-12-04T17:50:04.189+0900 I INDEX    [repl writer worker 1] 	 building index using bulk method; build may temporarily use up to 500 megabytes of RAM
2017-12-04T17:50:04.202+0900 I INDEX    [repl writer worker 1] build index done.  scanned 23 total records. 0 secs
